sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -337,6 +364,15 @@ def save_config(config):
     os.rename("{}.new".format(config_file), config_file)
 
 
+def get_auth_info(args):
```

Comment:
This should be underscored since it is a private function: `_get_auth_info`. If underscored, PyLint will prevent it being used outside this module.